main.py

```diff
@@ -76,118 +84,52 @@ async def on_ready(self):
         if not self.synced:
             await tree.sync(guild=discord.Object(id=aloraID))
             self.synced = True
-        logger.warning(f"Debug mode is on, remember about it")
+
+        # try:
+        #     cursor.execute("CREATE")
+        # except Error as e:
+        #     logger.error(f"{e}")
+        #     return None
+
+
+
+        if debugMode == True: logger.warning(f"Debug mode is on, remember about it")
         logger.info(f"We have logged in as {self.user}")
         logger.info(f"Servers:")
         for idx, guild in enumerate(self.guilds):
             logger.info(f"{idx+1}. {guild.name}({guild.id})")
-            # start_xp = 1000
-            # sum = start_xp
-            # idx = 1
+            for member in guild.members:
+                try:
+                    cursor.execute(f"UPDATE members SET voice_join_time = %s WHERE id = %s", (datetime.now(), member.id))
```

Comment:
https://planetscale.com/blog/what-is-n-1-query-problem-and-how-to-solve-it

You can use https://www.sqlalchemy.org/ for that, no need to write bad sql when someone created good one